diff --git a/src/OpcUaStackServer/Application/ApplicationIf.h b/src/OpcUaStackServer/Application/ApplicationIf.h
index 56c4f70..fd4e900 100644
--- a/src/OpcUaStackServer/Application/ApplicationIf.h
+++ b/src/OpcUaStackServer/Application/ApplicationIf.h
@@ -24,6 +24,7 @@
 #include "OpcUaStackCore/Certificate/CryptoManager.h"
 #include "OpcUaStackServer/Application/ApplicationServiceIf.h"
 #include "OpcUaStackServer/Application/ApplicationInfo.h"
+#include "OpcUaStackServer/InformationModel/InformationModel.h"
 
 namespace OpcUaStackServer
 {
@@ -80,9 +81,17 @@ namespace OpcUaStackServer
 		void applicationData(ApplicationData::SPtr& applicationData);
 		ApplicationData::SPtr& applicationData(void);
 
+		void informationModel(InformationModel::SPtr model) {
+		  informationModel_ = model;
+		}
+
+		InformationModel::SPtr informationModel(void) {
+		  return informationModel_;
+		}
+
 	  private:
 		ApplicationData::SPtr applicationData_;
-
+		InformationModel::SPtr informationModel_;
 	};
 
 }
